catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ImportCommand.java

```diff
@@ -308,7 +308,6 @@ public void close() throws IOException {
       byte[] buffer;
       while ((buffer = buf) != null) {
         if (BUF_UPDATER.compareAndSet(this, buffer, null)) {
-          InputStream input = in;
           in = null;
           // Purposely do not close `input`
```

Comment:
This should be changed to state `Purposely do not close 'in'`

Reply:
Changed